src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -459,6 +459,7 @@ static unique_ptr<Picture> _imageBuildHelper(SvgNode* node, float vx, float vy,
         auto syt = y * sy;
         Matrix m = {sx, 0, node->node.image.x - sxt, 0, sy, node->node.image.y - syt, 0, 0, 1};
         picture->transform(m);
+        picture->size(w, h);
```

Comment:
For svg image loading, size set is called twice. (451 line)
Scaling and size set are also duplicated internally.
I don't think we need a size set of lines 462, 451 because we already do the proper transform.
What do you think?

Reply:
Great point. I missed size is called twice. Removing both is enough. Thanks